src/screens/HealthRecordingsScreen.tsx

```diff
@@ -83,7 +92,16 @@ const HealthRecordingsScreen = () => {
     });
   }, [eventEmitter]);
 
-  // -----------------------
+  useFocusEffect(
```

Comment:
can you comment why you have this function?

Reply:
will update Readme